src/app/dataset-view/dataset.subscriptions.interface.ts

```diff
@@ -38,8 +43,9 @@ export interface DatasetHistoryUpdate {
     kind?: DatasetKind;
 }
 
+export type GraphNodeType = DatasetBasicsFragment & { metadata: CurrentSourceFetchUrlFragment };
```

Comment:
This type is quite confusing. It's not a node of the graph, rather it's a new dataset fragment that can be used to build the graph.

Maybe `DatasetLineageBasicsFragment`

Reply:
Also, think how this would grow, if we support Fetch steps that are not related to URLs.

Reply:
Renamed to `DatasetLineageBasicsFragment`.

If `metadata` will be grow, we can extend it like this ==>`DatasetBasicsFragment & { metadata: CurrentSourceFetchUrlFragment & SomethingElseFragment}` or create `MetadataFragment`